projects/core/src/occ/services/occ-endpoints.service.ts

```diff
@@ -50,6 +64,22 @@ export class OccEndpointsService {
   }
 
   /**
+   * Returns the value configured for a specific endpoint
+   *
+   * @param endpointKey the configuration key for the endpoint to return
+   */
+  getRawEndpointValue(endpoint: string, scope?: string): string {
+    if (!this.config?.backend?.occ) {
```

Comment:
Why we need to check this property here? It doesn't seem related to raw endpoint value.

Reply:
You are right